packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -518,19 +518,19 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       torrent.wires.map(async wire => {
         const {seedingChannelId, leechingChannelId} = wire.paidStreamingExtension;
         if (leechingChannelId) {
-          log.info(`About to close payment channel ${leechingChannelId}`);
-          await this.paymentChannelClient.getLatestPaymentReceipt(leechingChannelId);
-          await this.paymentChannelClient.closeChannel(leechingChannelId);
-          wire.paidStreamingExtension.leechingChannelId = null;
-          log.info(`Channel ${leechingChannelId} closed`);
+          log.info(`SP-> About to close payment channel ${leechingChannelId}`);
+          return this.paymentChannelClient.closeChannel(leechingChannelId, true).then(() => {
+            wire.paidStreamingExtension.leechingChannelId = null;
+            log.info(`SP-> Channel closed: ${leechingChannelId}`);
+          });
```

Comment:
Why the change from `await` to `.then`?

Reply:
I wanted to chain promises, but I'll make an extra function and use that instead, so that I maintain consistency.

Reply:
https://github.com/statechannels/monorepo/pull/1969/commits/034e40f790457b167989f3792ebeda37eaecdbb4